### ISSUE TITLE
Graphics needed for Mozfest email campaign

### ISSUE THREAD

**Issue by cassiemc** [mozilla/2015.mozillafestival.org#237]
@HPaulJohnson @KevZawacki Any deets on whether (and when) this is needed? We can probably repurpose graphics we create for the social shares #226, but we should confirm it'll work with your messaging.

**Comment by cassiemc** [comment]
Going to assign to @natalieworth but we'll still need more details before we can get started.

**Comment by cassiemc** [comment]
Ping @KevZawacki @HPaulJohnson Any updates on what's needed here?

**Comment by cassiemc** [comment]
@Saallen Last I heard this might not be necessary. Can you advise on if you need design support for the emails? It might be possible to repurpose the social share graphics from #226 if you think you might need just a simple, generic graphic.

**Comment by natalieworth** [comment]
@Saallen Are there any updates if this is needed still? Thanks

**Comment by cassiemc** [comment]
@Saallen @HPaulJohnson I'm going to close this ticket since we don't have any further details here. Please file a new ticket if needed!

**Comment by HPaulJohnson** [comment]
Works for me. As i understand it according to the DARCI, @Saallen is handling all the email communications.
